src/middlewares/routeActivation.ts

```diff
@@ -78,6 +78,7 @@ export default class RouteActivationMiddleware {
     );
   }
 
+  // eslint-disable-next-line sonarjs/no-identical-functions
```

Comment:
Why are we adding this ?

Reply:
there were two different functions which were identical but with different fn names in two different flows, so going with current implementation of maintaining two different fn names with same body to accommodate changes in 1 flow if comes later